README.md

```diff
@@ -100,7 +100,7 @@ Parameters: none
 Platform requirements:
 
 * [PHP] 8.0 or later (tested up to 8.1)
-* [MediaWiki] 1.35 or later (tested up to 1.39 and master)
+* [MediaWiki] 1.39 or later (tested up to 1.41 and master)
```

Comment:
We didn't drop 1.35 support: https://github.com/ProfessionalWiki/ExternalContent/blob/ea9185abdc2ad42faf267670d954afec3766e9ef/extension.json#L16

Reply:
Nice that it is working. Officially MW 1.35 to 1.38 do not support PHP 8.0 which is what I wanted to convey here.